README.md

```diff
@@ -169,6 +169,7 @@ Community Maintained Cars and Features
 | Nissan    | Leaf 2018-20                  | ProPILOT          | Stock            | 0mph               | 0mph         |
 | Nissan    | Rogue 2018-19                 | ProPILOT          | Stock            | 0mph               | 0mph         |
 | Nissan    | X-Trail 2017                  | ProPILOT          | Stock            | 0mph               | 0mph         |
+| Skoda     | Superb III B8 2015-2019       | Driver Assistance | Stock            | 0mph               | 0mph         |
```

Comment:
only put the years for which the port has been tested

Reply:
All done